pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/mail/MailTemplateRepeatableElement.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.ddp.model.mail;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.util.Map;
+
+@Value
+@AllArgsConstructor
+public class MailTemplateRepeatableElement {
+    String name;
+    String content;
+    Map<String, String> substitutions;
+
+    public String render() {
+        var renderedContent = String.valueOf(content);
+        for (final var substitution : substitutions.entrySet()) {
+            renderedContent = renderedContent.replace(toKey(substitution.getKey()), substitution.getValue());
+        }
```

Comment:
Extended `setSubstitutions` method's strem with following step:
`.removeBy(MailTemplateRepeatableElement::getName, null)`

It will remove all such repeatable elements that has null instead of name